### PR TITLE
VACMS 2700 single alert added below page intro blocks

### PR DESCRIPTION
**Pull request by kaise-lafrai**
## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/2700

## QA steps

As an admin

1. Login

2. Confirm the field placement and content creation experience matches the Acceptance Criteria below:
  - [x] new field "Alert" field_alert| entity revisions reference to "Alert (Single)" paragpraph only | cardinality single is added 
          to the following content types:
      - [x] FAQ - multiple Q&As (alert field located directly under "Title and introduction" fieldset)
      - [x] Checklist (alert field located directly under "Title and introduction" fieldset)
      - [x] Q&A (alert field located directly under "Question and Answer" fieldset)
      - [x] Step-by-step (alert field located directly under "Title and introduction" fieldset)
 - [x] node display configuration is adjusted to reflect the order of information on the form for each of the above content 
        types
 - [x] spec tool / content model tests are updated

3. Confirm Alert Single behaves as intended on each of the following content types node creation and edit:
- [x] FAQ
- [x] Checklist
- [x] Q&A
- [x] Step-by-step

## Definition of Done
- [ ] Product release are written (or in progress), if required.
- [ ] Documentation has been updated, if applicable.

### CODE REVIEW

**Comment by kaise-lafrai** [comment]
Ready for review.

File: config/sync/core.entity_form_display.node.checklist.default.yml [hunk 1]
```diff
@@ -54,7 +52,6 @@ third_party_settings:
     group_editorial_workflow:
       children:
         - moderation_state
-        - revision_log
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.node.checklist.default.yml:57]: This is one of those things that keeps getting lost through no fault of yours.

File: config/sync/core.entity_form_display.node.faq_multiple_q_a.default.yml [hunk 2]
```diff
@@ -68,9 +69,8 @@ third_party_settings:
     group_editorial_workflow:
       children:
         - moderation_state
-        - revision_log
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.node.faq_multiple_q_a.default.yml:71]: I think this needs to be put back.

File: config/sync/core.entity_form_display.node.q_a.default.yml [hunk 3]
```diff
@@ -21,9 +22,8 @@ third_party_settings:
     group_editorial_workflow:
       children:
         - moderation_state
-        - revision_log
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.node.q_a.default.yml:24]: gone missing needs to be put back.

File: config/sync/core.entity_form_display.node.step_by_step.default.yml [hunk 4]
```diff
@@ -51,9 +52,8 @@ third_party_settings:
     group_editorial_workflow:
       children:
         - moderation_state
-        - revision_log
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.node.step_by_step.default.yml:54]: Needs to be put back

**Comment by kaise-lafrai** [comment]
Rebased, resolved requested changes. Ready for review.

**Pull request merged**